Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -341,6 +342,7 @@ - (void)viewDidLoad {
                            @"Mini Form",
                            @"Optional Form",
                            @"Selection Survey",
+                           @"Predicate Tests",
```

Comment:
I'd move this test to the `Miscellaneous` section (along with the `Navigable Ordered Task` test). This is not testing the question steps themselves but the predicate matching logic. ;-)